packages/kbn-plugin-discovery/src/parse_kibana_platform_plugin.ts

```diff
@@ -38,6 +38,7 @@ export interface KibanaPlatformPluginManifest {
   };
   // TODO: make required.
   description?: string;
+  enableForAnonymousPages?: boolean;
```

Comment:
Maybe there are two parts to this. Letting the plugin owner decide if it should be available to anonymous pages and allowing a page to define which plugins it wants to render. The first allows domain experts to decide if their plugins is sensitive or not and the second lets page owners selectively decide what they render ?

Reply:
I think that's a good option yes, we add now the capability for plugins to opt-in for being used for anonymous pages, and later if required, we can let pages explicitly add additional plugins.

Reply:
> as the 'authenticated' side of Kibana is a full SPA where it would be not possible (at least which current architecture) to have different plugin trees per page/app.
Agreed and I suspect that we'll end up having to revisit that eventually when/if init times hit a critical point.